README.rst

```diff
@@ -0,0 +1,517 @@
+
+Ensemble Data Assimilation
+--------------------------
+
+Ensemble DA is a technique for combining observations with numerical models
+to estimate the state of a physical system.
+
+It enables modelers, observational scientists, and geophysicists to:
+
+- Generate initial conditions for forecasts.
+- Create a retrospective estimate of the state of a system, a practice known as
+  producing a *reanalysis*.
+- Assess the relative value of specific observations on forecast skill, a
+  practice known as conducting an *observing system experiment (OSE)*.
+- Estimate the value of hypothetical observations in order to inform the design
+  of an observing system, a practice known as conducting an *observing system
+  simulation experiment (OSSE)*. 
+- Determine a model's systematic bias in estimating the state of a system, a
+  practice known as diagnosing *model error*.
+
+The DART software environment makes it easy to explore a variety of data
+assimilation methods and observations with different numerical models. It
+provides powerful, flexible DA tools that are easy to use and customize to
+support efficient and reliable DA applications. While DART is primarily
+oriented for DA research, it has also been used in operational settings.
+
+DART includes:
+
+- A comprehensive tutorial introducing the concepts of ensemble DA.
+- Extensive documentation of its source code.
+- Interfaces to a variety of models and observation sets that can be used to
+  introduce new users or graduate students to ensemble DA.
+
+DART is also designed to facilitate the combination of assimilation algorithms,
+models, and real or synthetic observations to allow increased
+understanding of all three. It provides a framework for developing, testing,
+and distributing advances in ensemble DA to a broad community of users by
+removing the implementation-specific peculiarities of one-off DA systems.
+
+These tools are intended for use by the full range of geosciencies community:
+beginners and experts; students and teachers; national centers and university
+research labs.
+
+Organization of the documentation
+---------------------------------
+
+Because of DART's extensive scope, this documentation is detailed and
+carefully organized, enabling you to easily find the information you need. If
+you have any questions or suggestions for improvements, please contact DAReS
+staff by emailing dart@ucar.edu.
+
+The documentation is partitioned into three parts:
+
+- a user guide that explains how to install DART and perform data assimilation
+- source code documentation that provides a detailed description of the
+  programs and modules in the repository
+- a comprehensive description of data assimilation theory
+
+Manhattan Release
+-----------------
+
+The Manhattan release is new and currently supports only a subset of the
+models. DAReS staff will port over any requested model. Email dart@ucar.edu
+if yours is not on the list.
```

Comment:
here's a suggested rewrite of this paragraph:

```suggestion
DART releases are named based on the major version number.   The current version, 9.x.x,  is the Manhattan release. 
Email dart@ucar.edu for advice if you are interested in a model which has not been converted from the previous Lanai release.
```

i don't think we want to volunteer to convert any requested model, and i'm just avoiding too many details here.

Reply:
I agree. Nancy's wording is more realistic of the level of support we can provide.